### PR TITLE
Fix while (1) and while (0) to use TRUE/FALSE

### PR DESCRIPTION
**Pull request by Phlosioneer**

### CODE REVIEW

**Comment by garakmon** [comment]
whatever you used to do the replacing here messed up the encoding

File: src/contest_effect.c [hunk 1]
```diff
@@ -129,7 +129,7 @@ static void ContestEffect_UserLessEasilyStartled(void)
     SetContestantEffectStringID(shared192D0.contestant,CONTEST_STRING_STOPPED_CARING);
 }
 
-// Slightly startles the POK�MON in front.
+// Slightly startles the POKéMON in front.
```
Comment (Phlosioneer) [inline src/contest_effect.c]: I don't understand what the problem is here. Using a UTF-8 string analyzer, both `POKéMON` displayed here are using U+00E9 for the `e`.

**Comment by DizzyEggg** [comment]
@Diegoisawesome I don't really like this change tbh, while 1 is faster to write and read and it's the same thing anyway

**Comment by huderlem** [comment]
Yeah, I don't feel strongly enough to enforce this on the whole codebase, especially because it's not a typical C thing, to my knowledge.  Should probably just close this.

**Pull request closed without merging**